medicles/templates/medicles/detail.html

```diff
@@ -114,101 +113,202 @@
+            height: 100%;
+            display: -webkit-flex;
+            display: flex;
+            -webkit-justify-content: center;
+            justify-content: center;
+            -webkit-align-items: center;
+            align-items: center;
+            padding: 0 25px;
+            z-index: 5;
+            position: relative;
+        }
+
+        label:nth-child(1) .bg {
+            right: 0;
+            left: auto;
+        }
+
+        label:nth-child(2) .bg {
+            left: 0;
+        }
+
+        [type="radio"]:checked + .bg {
+            width: 100%;
         }
+
+
     </style>
 </head>
 
-<body>
+<body onload="getPreviousAnnotations()">
 
-    <div class="topnav" id="myTopnav">
-        <a class="active" href="{% url 'medicles:index' %}"><i class="fa fa-fw fa-home"></i> Home</a>
-        <a href="{% url 'medicles:advanced_search' %}"><i class="fa fa-fw fa-search"></i> Search</a>
-        {% if user.is_authenticated %}
+<div class="topnav" id="myTopnav">
+    <a class="active" href="{% url 'medicles:index' %}"><i class="fa fa-fw fa-home"></i> Home</a>
+    <a href="{% url 'medicles:index' %}"><i class="fa fa-fw fa-search"></i> Search</a>
+    {% if user.is_authenticated %}
         <a href="{% url 'medicles:user_search' %}">User Search</a>
+        <a>
+            Hi {{ user.username }}!
+            <a href="{% url 'medicles:logout' %}">Log Out</a>
+            {% else %}
+            <a href="{% url 'medicles:login' %}"><i class="fa fa-fw fa-user "></i> Login</a>
+            <a href="{% url 'medicles:signup' %}"><i class="fa fa-fw fa-address-book"></i> Signup</a>
+        </a>
+    {% endif %}
 
-      <a href="{% url 'medicles:logout' %}" style="float: right;">Log Out</a>
-    <a href="{% url 'medicles:profile' user.id%}" style="float: right;" >Hi {{ user.username }}!</a>
-      {% else %}
-      <a href="{% url 'medicles:login' %}" style="float: right;"><i class="fa fa-fw fa-user"></i> Login</a>
-      <a href="{% url 'medicles:signup' %}" style="float: right;"><i class="fa fa-fw fa-address-book"></i> Signup</a>
+    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+        <i class="fa fa-bars"></i>
+    </a>
+</div>
 
-        {% endif %}
+<div class="float-container">
 
-        <a href="javascript:void(0);" class="icon" onclick="myFunction()">
-            <i class="fa fa-bars"></i>
-        </a>
-    </div>
-
-    <div class="w3-margin">
+    <div class="float-child" id="article-division" style="width:fit-content">
         <h2 class="w3-container">Specific Article</h2>
 
         <div class="w3-container">
 
-            <h3 class="w3-text-teal"><a href="{% url 'medicles:detail' article.article_id %}">{{ article.article_title}}
+            <h3 class="w3-text-teal"><a
+                    href="{% url 'medicles:detail' article.article_id %}"></a>{{ article.article_title }}
             </h3>
-
-
-            <button style="float:right;" id="favourite" data-id="{{ article.article_id }}"
```

Comment:
This button is to favorite the article. We should not remove this part.

Reply:
I added this part again